llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -9004,6 +8988,20 @@ bool VPRecipeBuilder::getScaledReductions(
   return false;
 }
 
+SmallVector<std::pair<unsigned, MDNode *>>
+VPRecipeBuilder::getMetadataToPropagate(Instruction *I) const {
+  SmallVector<std::pair<unsigned, MDNode *>> Metadata;
+  ::getMetadataToPropagate(I, Metadata);
```

Comment:
Would be nice if both getMetaDataToPropagate functions have the same signature: take Metadata as second argument.